recipes/chaospy/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "2.2.4" %}
+
+package:
+  name: chaospy
+  version: {{ version }}
+
+source:
+  fn: chaospy-{{ version }}.tar.gz
+  url: https://github.com/jonathf/chaospy/archive/v{{ version }}.tar.gz
+  sha256: b8c198d9e1139de494dcb1449945d25635721f3cbdd0ca27588ea6000ae4d9ed
+
+build:
+  number: 0
+  noarch: python
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - networkx
+    - scikit-learn
+
+test:
+  imports:
+    - chaospy
+  source_files:
+    - tests
+  requires:
+    - pytest-runner
+  commands:
+    - py.test
+
+about:
+  home: https://github.com/jonathf/chaospy
+  license: BSD
```

Comment:
Should be BSD-3-Clause, and `license_family: BSD`.

Reply:
ok